### ISSUE TITLE
Add unit tests

### ISSUE THREAD

**Issue by jl5000** [jl5000/barRacer#3]
To test for different types of data

**Comment by Pilipino** [comment]
Hi Sir @jl5000! What kind of Unit Tests, would you suggest an implementation using `testthat`?

Thanks! :)

**Comment by jl5000** [comment]
Absolutely! The primary tests I was thinking of was to test that it can handle different types of time data (numeric, date, datetime) as I think it's only set up for years at the moment.

**Comment by Pilipino** [comment]
Hi Sir! Do you have recommended datasets that have datetime data Sir? :)

**Comment by jl5000** [comment]
Not yet :)
